### PR TITLE
Added features:Ursa.ReactiveUIExtension

### PR DESCRIPTION
**Pull request by WCKYWCKF**

### CODE REVIEW

File: src/Ursa.ReactiveUI/Ursa.ReactiveUIExtension.csproj [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		<Authors>WCKFWCKY</Authors>
+		<PackageId>Irihi.Ursa.ReactiveUIExtension</PackageId>
+		<PackageIcon>irihi.png</PackageIcon>
+		<PackageProjectUrl>https://github.com/irihitech/Ursa.Avalonia</PackageProjectUrl>
+		<PackageLicenseExpression>MIT</PackageLicenseExpression>
+		<Nullable>enable</Nullable>
+		<GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+		<Description>This is a Ursa expansion. This package integrates and is compatible with UrsaWindow and UrsaView with Avalonia.ReactiveUI. [Avalonia.ReactiveU See: https://docs.avaloniaui.net/zh-Hans/docs/concepts/reactiveui/]
+
+这个是一个Ursa拓展包。这个包整合并互相兼容了UrsaWindow和UrsaView与Avalonia.ReactiveUI的功能。【Avalonia.ReactiveU参见：https://docs.avaloniaui.net/docs/concepts/reactiveui/】</Description>
```
Comment (rabbitism) [inline src/Ursa.ReactiveUI/Ursa.ReactiveUIExtension.csproj]: ReactiveU=>ReactiveUI

File: src/Ursa.ReactiveUI/Ursa.ReactiveUIExtension.csproj [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		<Version>1.0.1</Version>
+		<Copyright>WCKFWCKY</Copyright>
+		<RepositoryUrl>https://github.com/irihitech/Ursa.Avalonia</RepositoryUrl>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Avalonia.ReactiveUI" Version="11.1.1" />
+		<PackageReference Include="Irihi.Ursa" Version="1.3.0" />
```
Comment (rabbitism) [inline src/Ursa.ReactiveUI/Ursa.ReactiveUIExtension.csproj]: You should include this project in the solution, directly reference Ursa project, and remove this package reference.

File: src/Ursa.ReactiveUI/Ursa.ReactiveUIExtension.csproj [hunk 3]
```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>netstandard2.0;net8</TargetFrameworks>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<LangVersion>latest</LangVersion>
+		<Authors>WCKFWCKY</Authors>
```
Comment (rabbitism) [inline src/Ursa.ReactiveUI/Ursa.ReactiveUIExtension.csproj]: If we can co-author like WCKFWCKY, IRIHI Technology? ( you can ignore if you don't want to)

**Comment by WCKYWCKF** [comment]
1. Fixed the bug of package description.
2. Change the package reference to a project reference.
3. Change the author item to a more appropriate description.

**Review by rabbitism — APPROVED** [review]

**Pull request merged**